statsService/src/main/R/JdbcHandler.R

```diff
@@ -285,7 +285,13 @@ generateTableIndexSQLQuery <- function(tableName, columnName) {
 
 convertSqlNansToNulls <- function(col) {
 
-	protectedCol <- paste("CASE WHEN", col, " = 'NAN' THEN NULL ELSE", col, "END")
+# If the tmp table contains NAN/NAs the datatype will be a VARCHAR so it needs to be cast to NUMERIC
```

Comment:
This seems wrong: a `VARCHAR` is by definition not a number. `NaN` is, in my understanding, a special value that _numeric columns can hold -- or at least return -- when they've received incorrect data.

Reply:
When R dumps the frame to the tmp table the column comes a VARCHAR with NA/NAN (as text) in it! This is the route cause of the problem. Postgres is a bit more picky about casting at this point as the table may contains numbers or text.